client/src/legacy/LeftAndMain.Tree.js

```diff
@@ -298,7 +298,9 @@ $.entwine('ss.tree', function($){
           }
           // Substitute html from request for that generated by jstree
           node.addClass(origClasses).html(newNode.html());
-          callback(node);
+          if (callback) {
+            callback(node);
+          }
```

Comment:
Not strictly necessary now I’ve re-added a callback, but may as well be defensive